### ISSUE TITLE
missing em_connector_push table

### ISSUE THREAD

**Issue by mdkashifmirza** [texasnick/cloudwatch-listener#1]
When I try to preview your update set in my personal instance, I am getting an error "Could not find a table em_connector_push this update requires"

**Comment by texasnick** [comment]
Do you have the event management plug-in enabled?

**Comment by mdkashifmirza** [comment]
Yes I do have event management Plugin enabled. its my Personal instance running Quebec release.


![Screenshot 2021](https://user-images.githubusercontent.com/62132992/113013628-7e4e0c00-9199-11eb-84e0-7f3723f512dc.png)

**Comment by texasnick** [comment]
Can you confirm if this table is in your system or not?  If not, you might try to repair the plugin installation.

![image](https://user-images.githubusercontent.com/10350818/113014801-875fbe80-9142-11eb-95f0-2b63f755343d.png)

**Comment by texasnick** [comment]
Actually, it appears in Quebec this table/feature has been deprecated.  There's nothing in the release notes about it, but I too have a new Quebec instance and it does not have that table either.  The docs note the need to migrate these transform scripts to new push connectors, but that would require an instance pre-Q to even load the update set.  https://docs.servicenow.com/bundle/quebec-it-operations-management/page/product/event-management/concept/migrate-transform-scripts.html
